.github/workflows/build.yml

```diff
@@ -1,59 +1,75 @@
-name: Build
+name: PR Checks
```

Comment:
Let's name it: Build & Test?

Reply:
Wasn't sure what to use tbh, because it is not just building and running tests. It does build, some go checks, lint checks and then tests.

Reply:
Hmm, Services also have it named that way `PR Checks`.

Reply:
Ok. Let's leave it as it is.